roles/network_plugin/flannel/templates/cni-flannel.yml.j2

```diff
@@ -0,0 +1,113 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-flannel-cfg
+  namespace: "{{system_namespace}}"
+  labels:
+    tier: node
+    app: flannel
+data:
+  cni-conf.json: |
+    {
+      "name": "cbr0",
+      "type": "flannel",
+      "delegate": {
+        "forceAddress": true,
+        "hairpinMode": true,
+        "isDefaultGateway": true
+      }
+    }
+  net-conf.json: |
+    {
+      "Network": "{{ kube_pods_subnet }}",
+      "Backend": {
+        "Type": "{{ flannel_backend_type }}"
+      }
+    }
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube-flannel
+  namespace: "{{system_namespace}}"
+  labels:
+    tier: node
+    k8s-app: flannel
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        k8s-app: flannel
+    spec:
+{% if rbac_enabled %}
+      serviceAccountName: flannel
+{% endif %}
+      containers:
```

Comment:
Insert this block: 
```
      updateStrategy:
        type: RollingUpdate
```

Reply:
`updateStrategy: type: RollingUpdate` should already be in there. Bottom of the file.

Reply:
I'm not sure if it's in the right spot, but it's easy enough to test later